kubernetes/samples/scripts/create-weblogic-domain/model-in-image/sample_app/wlsdeploy/applications/sample_war/WEB-INF/weblogic.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+
+<weblogic-web-app xmlns="http://www.bea.com/ns/weblogic/90">
+  <session-descriptor>
+     <timeout-secs>15</timeout-secs>
+     <invalidation-interval-secs>60</invalidation-interval-secs>
+  </session-descriptor>
+  <jsp-descriptor>
+    <page-check-seconds>1</page-check-seconds>
+    <verbose>true</verbose>
+  </jsp-descriptor>
+    <!--

```

Comment:
Why is this commented?

Reply:
Removed from sample descriptor.

Reply:
@jshum2479 I still see the comment.  Please explain why it needs to remain, or remove it and resolve this comment.

Reply:
already removed